components/button/events.md

```diff
@@ -18,20 +18,42 @@ This article explains the events available in the Telerik Button for Blazor:
 
 The `OnClick` event fires when the user clicks or taps the button.
 
+It receives argument of type `MouseEventArgs`  which exposes the following fields:
```

Comment:
There is no need to list these options here, link MSDN - it is a framework class.

Reply:
Done